docs/pages/api/radio.md

```diff
@@ -35,10 +35,10 @@ You can learn more about the difference by [reading our guide](/guides/minimizin
 | <span class="prop-name">inputProps</span> | <span class="prop-type">object</span> |  | [Attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Attributes) applied to the `input` element. |
 | <span class="prop-name">inputRef</span> | <span class="prop-type">func<br>&#124;&nbsp;object</span> |  | This prop can be used to pass a ref callback to the `input` element. |
 | <span class="prop-name">name</span> | <span class="prop-type">string</span> |  | Name attribute of the `input` element. |
-| <span class="prop-name">onChange</span> | <span class="prop-type">func</span> |  | Callback fired when the state is changed.<br><br>**Signature:**<br>`function(event: object, checked: boolean) => void`<br>*event:* The event source of the callback. You can pull out the new value by accessing `event.target.value`.<br>*checked:* The `checked` value of the switch |
+| <span class="prop-name">onChange</span> | <span class="prop-type">func</span> |  | Callback fired when the state is changed.<br><br>**Signature:**<br>`function(event: object) => void`<br>*event:* The event source of the callback. You can pull out the new value by accessing `event.target.value` (string). You can pull out the new checked state by accessing `event.target.checked` (boolean). |
```

Comment:
The "new value", you say - but does onChange fire when the value changes, or only when the checked state changes? If the latter, then probably the value isn't relevant in this context.

Reply:
It depends, we could consider that the checked state is not relevant either, it will always be true when the change event fire. I think that it depends on how people handle the state of the form.

Reply:
> I think that it depends on how people handle the state of the form.

Sure

> we could consider that the checked state is not relevant either, it will always be true when the change event fire.

Wait, what? Doesn't the event fire when it's toggled off?

Reply:
> Wait, what? Doesn't the event fire when it's toggled off?

No, not at the input radio level.

Reply:
Gotcha 👍